src/c-writer.cc

```diff
@@ -1910,6 +1911,39 @@ void CWriter::WriteInitInstanceImport() {
   Write(CloseBrace(), Newline());
 }
 
+void CWriter::WriteImportProperties(CWriterPhase kind) {
+  if (import_module_set_.empty())
+    return;
+
+  auto write_import_prop = [&](const Import* import, std::string prop,
+                               uint32_t value) {
+    Write(Newline());
+    if (kind == CWriterPhase::Declarations) {
+      Write("extern ");
+    }
+    Write("const u32 " + module_prefix_ + "_",
+          prop + "_" + MangleName(import->module_name) +
+              MangleName(import->field_name));
+    if (kind == CWriterPhase::Definitions) {
+      Write(" = ", value);
+    }
+
+    Write(";", Newline());
+  };
+
+  for (const Import* import : unique_imports_) {
+    if (import->kind() == ExternalKind::Memory) {
+      const Limits* memory_limits =
+          &(cast<MemoryImport>(import)->memory.page_limits);
+      write_import_prop(import, "initial", memory_limits->initial);
```

Comment:
Could we maybe call this "min" to match the WebAssembly term? (https://webassembly.github.io/spec/core/syntax/types.html#syntax-limits)

Also, would be nice to publish this for tables as well, but that can be another PR.

Reply:
Will change to min :+1: